ios/moviesreloadedMiniApp/ErnRunner.xcodeproj/project.pbxproj

```diff
@@ -171,6 +171,7 @@
 				TargetAttributes = {
 					304F3E941EEF455C00F1E36B = {
 						CreatedOnToolsVersion = 8.3.2;
+						DevelopmentTeam = 4F9HS269YY;
```

Comment:
What's this development team for, and do we need it? I don't know much about Xcode, but I've seen this a few times before. If I remember correctly, it resulted in the project not being usable in Xcode without some preliminary setup steps (i.e. adding a specific team/account or something). So just checking...